flexeval/core/language_model/openai_api.py

```diff
@@ -91,6 +96,9 @@ def __init__(
         model_limit_new_tokens: int | None = None,
         max_parallel_requests: int | None = None,
         tools: list[dict[str, Any]] | None = None,
+        max_num_trials: int | None = None,
+        first_wait_time: int | None = None,
+        max_wait_time: int | None = None,
```

Comment:
なるべくユーザーに近い側ででデフォルト値を指定しまった方が、コードが読みやすくなると思いました。
そうでないと `_retry_on_error` まで潜っていくことにことになるためです。

```suggestion
        max_num_trials: int = 5,
        first_wait_time: int = 10,
        max_wait_time: int = 80,
```